go.mod

```diff
@@ -1,28 +1,25 @@
 module github.com/drand/kyber
 
-go 1.18
+go 1.25
```

Comment:
Could we have the lowest version as required by dependencies rather than the latest one?
I suspect go 1.23 or  go 1.24 should suffice.